win/sassc.vcxproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <SASSC_VERSION>[NA]</SASSC_VERSION>
     <LIBSASS_VERSION>[NA]</LIBSASS_VERSION>
-    <LIBSASS_DIR>..\..</LIBSASS_DIR>
+    <LIBSASS_DIR Condition="'$(LIBSASS_DIR)' == ''">..\..</LIBSASS_DIR>
```

Comment:
Is this condition really needed? My tests indicated that it can still be overloaded by command line arg `/P:LIBSASS_DIR=...`. Or is this related to environment variables?